server/shared/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -22,125 +22,12 @@ import cats.Applicative
 import cats.Monad
 import cats.data.Kleisli
 import cats.syntax.all._
-import org.http4s.Method.OPTIONS
 import org.http4s.headers._
 import org.http4s.syntax.header._
-import org.log4s.getLogger
 import org.typelevel.ci._
+import org.typelevel.log4cats.LoggerFactory
 
-import scala.annotation.nowarn
 import scala.concurrent.duration._
-import scala.util.hashing.MurmurHash3
-
-/** CORS middleware config options.
-  * You can give an instance of this class to the CORS middleware,
-  * to specify its behavior
-  */
-
-@deprecated(
-  """Deficient. See https://github.com/http4s/http4s/security/advisories/GHSA-52cf-226f-rhr6.""",
-  "0.21.27",
-)
```

Comment:
I removed all the deprecated CORS stuff so I wouldn't have to rework its logging.